src/__tests__/convert-spec.js

```diff
@@ -0,0 +1,62 @@
+import {convertContentTo, convertContentFrom} from '../lib/convert';
+import DraftJS, {ContentState} from 'draft-js';
+
+describe('convertContentTo', () => {
+
+	let draftJsContent;
+	beforeEach(() => {
+		draftJsContent = DraftJS.ContentState.createFromText('<div>some <a href="#">content</a></div>');
+	});
+
+	it('converts DraftJS content to json', () => {
+		expect(draftJsContent).toBeInstanceOf(ContentState);
+		let content = convertContentTo(draftJsContent, 'json');
+		expect(content).toBeInstanceOf(Object);
+	});
+
+	it('converts DraftJS content to html string', () => {
+		expect(draftJsContent).toBeInstanceOf(ContentState);
+		let content = convertContentTo(draftJsContent, 'html');
+		expect(typeof content).toBe('string');
```

Comment:
If we are converting to HTML, don't we want to compare the result with an actual HTML type instead of string?

Reply:
it converts it to a string, not actual HTML

Reply:
It's a bit confusing though, what;s would be the difference then between converting to html or converting to text?

Reply:
passing in 'text' as type strips html tags. Convert to html returns an escaped html string.